autoload/neomake/makers/proselint.vim

```diff
@@ -0,0 +1,7 @@
+" vim: ts=4 sw=4 et
+
+function! neomake#makers#proselint#proselint() abort
```

Comment:
good point, I have moved it in [b9482a5](https://github.com/neomake/neomake/pull/775/commits/b9482a5fd493aa29859c21ddfc75452257ed604c)